build/AzurePipelinesTemplates/MUX-CreateNugetPackage-Job.yml

```diff
@@ -8,6 +8,7 @@ parameters:
   primaryBuildArch: x86
   buildFlavor: Release
   signConfig: ''
+  releasePackage: 'false'
```

Comment:
maybe officalRelease or just usePrereleaseTag are better than this name.
Another option is export prereleaseTag directly, and default is "prerelease".

Reply:
Fair enough, i'll change it to "usePrereleaseTag".

Reply:
Actually one of the problems I had was manipulating the queue-time variable at this level. The flag needs to be the opposite -- how about noPrereleaseTag?

Reply:
good

Reply:
or maybe just add useReleasePackageTag?

Reply:
useReleasePackageTag and with the default setting opposite